RUFAS/biophysical/manure/separators/separator.py

```diff
@@ -0,0 +1,133 @@
+from dataclasses import dataclass
+from RUFAS.biophysical.manure.processor import Processor
+from RUFAS.current_day_conditions import CurrentDayConditions
+from RUFAS.data_structures.animal_to_manure_connection import ManureStream
+from RUFAS.time import Time
+
+
+@dataclass
+class SeparatorConfig:
+    """
+    Base configuration for a manure separator.
+
+    Attributes
+    ----------
+    water_efficiency : float
+        The efficiency of the separator in removing water from the manure.
+    ammoniacal_nitrogen_efficiency : float
+        The efficiency of the separator in removing ammoniacal nitrogen from the manure.
+    nitrogen_efficiency : float
+        The efficiency of the separator in removing nitrogen from the manure.
+    phosphorus_efficiency : float
+        The efficiency of the separator in removing phosphorus from the manure.
+    potassium_efficiency : float
+        The efficiency of the separator in removing potassium from the manure.
+    ash_efficiency : float
+        The efficiency of the separator in removing ash from the manure.
+    non_degradable_volatile_solids_efficiency : float
+        The efficiency of the separator in removing non-degradable volatile solids from the manure.
+    degradable_volatile_solids_efficiency : float
+        The efficiency of the separator in removing degradable volatile solids from the manure.
```

Comment:
```suggestion
    total_volatile_solids_efficiency : float
        The efficiency of the separator in removing volatile solids from the manure.
```
Same comment as in machine_processor.py. I think there are some other instances in this file that should be updated as well.